src/jit/namedintrinsiclist.h

```diff
@@ -88,6 +88,10 @@ enum HWIntrinsicFlag : unsigned int
     // NoJmpTable IMM
     // the imm intrinsic does not need jumptable fallback when it gets non-const argument
     HW_Flag_NoJmpTableIMM = 0x2000,
+
+    // No Read/Modify/Write Semantics
+    // the intrinsic does not have read/modify/write semantics and doesn't need
+    HW_Flag_NoRMWSemantics = 0x4000,
```

Comment:
I am not a fan of all these negative flags, as it is always confusing when you are checking for the positive case by checking the negation of negative case. But I see there's a lot of precedent for that in these flags already.

Reply:
I'll log an issue tracking an investigation towards fixing this.